app/helpers/application_helper.rb

```diff
@@ -98,7 +98,7 @@ def column_sort_icon
   end
 
   def dom_id(record, prefix = nil)
-    result = ActionView::RecordIdentifier.dom_id(record, prefix)
+    result = ActionView::RecordIdentifier.dom_id(record, prefix).dup
```

Comment:
just info: `dup` is clearer but you can also use `+ActionView::RecordIdentifier.dom_id(record, prefix)` to unfreeze